File: app/src/main/java/com/example/birdify/ui/Sightings/SightingsFragment.java
package com.example.birdify.ui.Sightings;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.lifecycle.ViewModelProvider;


import com.example.birdify.databinding.FragmentSightingsBinding;

public class SightingsFragment extends Fragment {

    private FragmentSightingsBinding binding;

    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        SightingsViewModel sightingsViewModel =
                new ViewModelProvider(this).get(SightingsViewModel.class);

        binding = FragmentSightingsBinding.inflate(inflater, container, false);
        View root = binding.getRoot();



        return root;
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding = null;
    }
}

File: app/src/main/java/com/example/birdify/ui/Sightings/SightingsFragment.kt
package com.example.birdify.ui.Sightings

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.lifecycle.ViewModelProvider
import com.example.birdify.databinding.FragmentSightingsBinding

class SightingsFragment : Fragment() {
    private var binding: FragmentSightingsBinding? = null
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?, savedInstanceState: Bundle?
    ): View? {
        val sightingsViewModel =
            ViewModelProvider(this).get(
                SightingsViewModel::class.java
            )
        binding = FragmentSightingsBinding.inflate(
            inflater,
            container,
            false
        )
        return binding!!.root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        binding = null
    }
}